src/python/CMakeLists.txt

```diff
@@ -0,0 +1,78 @@
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Do we want to put `python` dir under `src`?

Reply:
For simplicity of this PR, `python` was added under `src`. Moving it out of `src` could potentially be a follow up PR?

Reply:
DLIS-7071